apps/next/src/components/FullscreenModal/ModalPageControlProvider.tsx

```diff
@@ -48,8 +48,8 @@ export const ModalPageControlProvider: FC<{ children: ReactNode }> = ({
   );
 
   useEffect(() => {
-    setIsBackButtonVisible(current > 1);
```

Comment:
With this change even on the first page of the modal the back button is visible:
<img width="619" height="738" alt="Screenshot 2025-07-21 at 9 35 33" src="https://github.com/user-attachments/assets/bdbaa589-7b84-4c26-9499-cec91a18993f" />